music21/metadata/__init__.py

```diff
@@ -2605,6 +2554,185 @@ def update(self, streamObj):
                                     pitchHighest=self.pitchHighest,
                                     )
 
+    def all(self, skipContributors: bool = False) -> t.Tuple[t.Tuple[str, t.Any], ...]:
+        '''
+        Returns all values stored in this RichMetadata as a sorted Tuple of Tuples.
+        Each individual Metadata Tuple is (uniqueName, strValue) and each additional
+        RichMetadata tuple is (uniqueName, richAttributeValue).
+
+        >>> rmd = metadata.RichMetadata()
+        >>> c = corpus.parse('corelli/opus3no1/1grave')
+        >>> rmd.merge(c.metadata)
+        >>> rmd.update(c)
+        >>> rmd.all()
+        (('ambitus',
+            AmbitusShort(semitones=48, diatonic='P1', pitchLowest='C2', pitchHighest='C6')),
+         ('arranger', 'Michael Scott Cuthbert'),
+         ('composer', 'Arcangelo Corelli'),
+         ('copyright', '© 2014, Creative Commons License (CC-BY)'),
+         ('fileFormat', 'musicxml'),
+         ('filePath', '...corpus/corelli/opus3no1/1grave.xml'),
+         ('keySignatureFirst', -1),
+         ('keySignatures', [-1]),
+         ('movementName', 'Sonata da Chiesa, No. I (opus 3, no. 1)'),
+         ('noteCount', 259),
+         ('numberOfParts', 3),
+         ('pitchHighest', 'C6'),
+         ('pitchLowest', 'C2'),
+         ('quarterLength', 76.0),
+         ('software', 'Dolet Light for Finale 2014'),
+         ('software', 'Finale 2014 for Mac'),
+         ('software', 'music21 v.8.0.0a9'),
+         ('sourcePath', 'corelli/opus3no1/1grave.xml'),
+         ('tempoFirst', None), ('tempos', []),
+         ('timeSignatureFirst', '4/4'),
+         ('timeSignatures', ['4/4']))
+        >>> rmd.dateCreated = metadata.DateRelative('1689', 'onOrBefore')
+        >>> rmd.localeOfComposition = 'Rome'
+        >>> rmd.all(skipContributors=True)
+        (('ambitus',
+            AmbitusShort(semitones=48, diatonic='P1', pitchLowest='C2', pitchHighest='C6')),
+         ('copyright', '© 2014, Creative Commons License (CC-BY)'),
+         ('dateCreated', '1689/--/-- or earlier'),
+         ('fileFormat', 'musicxml'),
+         ('filePath', '...corpus/corelli/opus3no1/1grave.xml'),
+         ('keySignatureFirst', -1),
+         ('keySignatures', [-1]),
+         ('localeOfComposition', 'Rome'),
+         ('movementName',
+         'Sonata da Chiesa, No. I (opus 3, no. 1)'),
+         ('noteCount', 259),
+         ('numberOfParts', 3),
+         ('pitchHighest', 'C6'),
+         ('pitchLowest', 'C2'),
```

Comment:
all this doesn't need to be repeated just to show one thing has changed.  use `...` to skip identical lines.

Reply:
Agreed.

Reply:
Done.